src/server/services/EmailService.ts

```diff
@@ -51,4 +52,27 @@ export default class EmailService {
         },
       });
   }
+
+  /**
+   * Sends an email with a link to confirm the user's account.
+   * @param request The web request associated with the email.
+   * @param account The account associated with the request.
+   * @param event The event associated with the request.
+   */
+  async sendEventAcceptanceEmail(request: Request, admin: Admin, event: TESCEvent, userEmail: string) {
+    Logger.info(`Sending acceptance email to ${userEmail} for event ${event.alias} by ${admin.username}`);
+
+    const ACCEPTANCE_EMAIL_TEMPLATE_ID = process.env.SENDGRID_ACCEPTANCE_EMAIL_ID;
```

Comment:
Load this from Config

Reply:
I don't want to add a config file to the project because it would add another gitignore'd file which makes setting up the codebase a bit more of a pain - why not just have it in .env?

Reply:
I meant we have a `~/Config/index.ts` file that loads in from `process.env` and places it into an object. This way we don't have random accesses to `process.env` elsewhere in the project.

Reply:
Woah, nice catch - Didn't notice we did.